contracts/Core/parameters/child/StakeManagerParams.sol

```diff
@@ -69,6 +70,12 @@ abstract contract StakeManagerParams is ACL, IStakeManagerParams, Constants {
         minStake = _minStake;
     }
 
+    function setMinSafeRazor(uint256 _minSafeRazor) external override onlyRole(GOVERNANCE_ROLE) {
+        require(_minSafeRazor <= minStake, "Invalid minSafeRazor Update");
```

Comment:
"Invalid minSafeRazor Update" -> Be more specific. Why is it invalid?

Reply:
The thing is minSafeRazor should not be be greater than minStake and @hrishikeshio  mentioned above to add this require here to be on the safer side . @SkandaBhat

Reply:
Yeah. I meant that the log must be more specific. 

"Invalid minSafeRazor Update" -> This is not conveying much information. Please add a better reason in the code.

Reply:
hey is this resolved ?

Reply:
Done @abhishekvispute @SkandaBhat f1541a6e51d507699d17eaa41c7069a5b951ad4e

Reply:
lint failed @shekhar2807 
Please fix

Reply:
@SkandaBhat @abhishekvispute changing the reason to "minSafeRazor must be less than minStake" is failing the lint due to big revert message . So what should be appropriate small message ?

Reply:
@SkandaBhat @abhishekvispute "minSafeRazor beyond minStake" makes sense? It would let build pass.

Reply:
Ya